src/styles/forms/_inputs.scss

```diff
@@ -51,6 +51,7 @@
 .form-control {
   &:not([type="checkbox"]):not([type="radio"]) {
     display: block;
+    width: 100%;
```

Comment:
Just checking this is the right place to add it, we intend not to do this for checkbox and radio?